cvat/apps/lambda_manager/views.py

```diff
@@ -138,6 +144,10 @@ def to_dict(self):
             response.update({
                 'state': self.state
             })
+        if self.kind is LambdaType.DETECTOR:
+            response.update({
+                'attributes': self.attributes
```

Comment:
@mikhail-treskin , Probably it is slightly better to return attributes together with labels.

Reply:
In general I agree but I was afraid that response structure changing  may broke compatibility between BE and FE in some cases which I'm not aware.